.travis.yml

```diff
@@ -45,7 +45,7 @@ script:
   # Install Janus
   - curl -sL https://raw.githubusercontent.com/ethereumproject/janus/master/get.sh | bash
   - export PATH=$PATH:$PWD/janusbin
-  - export APP_VERSION_GIT_TAG="$(janus version -format TAG_OR_NIGHTLY)"
+  - export APP_VERSION_GIT_TAG="$(janus version -format 'v%M.%m.%C-%S')"
```

Comment:
Also i'm thinking about `v%M.%m.%B` where B is `%P * 100 + %C`. So for default tag like `3.5.0` it gives same version as we have currenlty, but you can always fast forward `+100 -
 commits_count`. 
For example after commit 54 on tag 3.5.0 (gives v3.5.54) you can make a tag 3.5.1 and commit 4 (which is commit 58 == 54 after 3.5.0 + 4 after 3.5.1) you'll get version 3.5.104:
 - push 54 commits after tag 3.5.0 
 - you have version 3.5.54
 - tag this commit 3.5.1
 - push 4 more commits (it's 58 commits after tag 3.5.0)
 - get version 3.5.104